recipes/tiledb/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: tiledb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB/archive/{{ version }}.tar.gz
+  sha256: 6e613d222c5f15537dbbd0cf121138c27d480780cc56acd5a7d1ff9f9ce0fe56
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.2
+    - zlib 1.2.11
+    - bzip2
+    - zstd
+    - lz4-c 
+    - blosc 1.12.0
+  run:
+    - zlib 1.2.11
+    - bzip2
+    - zstd
+    - lz4-c
+    - blosc 1.12.0
+
+test:
+  commands:
+    - test -e $PREFIX/include/tiledb/tiledb.h  # [unix]
+    - test -e $PREFIX/include/tiledb/tiledb  # [unix]
+    - test -e $PREFIX/lib/libtiledb.a  # [unix]
+    - test -e $PREFIX/lib/libtiledb.so  # [linux]
```

Comment:
Doesn't matter, but FYI this line and the next could be replaced by <code>test -e $PREFIX/lib/libtiledb$SHLIB_EXT &nbsp;# [unix]</code>.